doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md

```diff
@@ -1,4 +1,6 @@
 #<a name="top"></a>Grouping Rules
+**IMPORTANT NOTE: from release 1.6.0, this feature is deprecated in favour of Name Mappings.**
```

Comment:
A link to the deprecated_and_removed.md file, to the place were this deprecation is explained, could help.